code/configurations/configurations.json

```diff
@@ -9,8 +9,8 @@
     "traefik": "traefik.miles.co.ke"
   },
   "ips": {
-    "default": "34.122.239.158",
-    "local": "10.128.0.2"
+    "default": " ",
+    "local": " "
```

Comment:
Why has this change been made?

Reply:
The local and default IPs shouldn't be specific? It should be dependent on the server we are using. So that's why I made changes to it, since every user must have different IPs or so.

Reply:
> The local and default IPs shouldn't be specific? It should be dependent on the server we are using. So that's why I made changes to it, since every user must have different IPs or so.

Do I need to still change it?

Reply:
Nope. It's fine.